### DISCUSSION TITLE
Should KeeperDAO txs be counted as MEV?

### DISCUSSION THREAD

**Discussion by obadiaa** [flashbots/mev-explore-public#14] [General]
Thanks to Ankur for surfacing this discussion.

This address 0x3d71d79c224998e608d03c5ec9b405e7a38505f0 is KeeperDAO. 
There is a debate as to whether KeeperDAO is MEV.

Here's a good example of KeeperDAO tx:
https://etherscan.io/tx/0x8d36d40e80948d867fe60240498f11d865cb805cb4d56f6f2d004193cca8522c

It looks like 16-17eth profit on this arb but the critical leg of the trade (3,694.406 AMPL for 19.4442 ETH) goes through 0x proxy with a specified tx.origin address. So only 0x3d71d79c224998e608d03c5ec9b405e7a38505f0 will ever win these and gas price is always low because its not a publicly accessible arb opportunity.

Why it isn't MEV:
KeeperDAO isn't MEV. They use 0x v4 RFQ-based orders with a defined tx.origin. These "arb opportunities" are only takeable by whitelisted addresses. It's not MEV if it's not extractable by a miner, as in it's not "miner extractable" value. The miner won't see any of that arb profit, and there's no open competition (gas war, etc) for that profit.

Why it is MEV:
Cartel MEV is still MEV. 

This bring up a broader question but if value related to transaction ordering is not permissionless extractable, aka is 'fenced', does it still fall under MEV?

What do you think?

**Comment by sambacha** [comment]
![](https://i.imgflip.com/4z3qxj.jpg)
